web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -114,7 +114,8 @@ interface UnifiedResourcesProps {
   //TODO(gzdunek): the pin button should be moved to some other place
   //according to the new designs
   Header(pinAllButton: React.ReactElement): React.ReactElement;
-  EmptySearchResults: React.ReactElement;
+  /** Rendered when resource list and search query are empty. */
```

Comment:
Ah, I ran into this recently with the Connect My Computer CTA when there's no resources.

I'd say "Typically used to inform the user that there's no matching resources when they want to list resources without filtering the list with a search query. Rendered only when the resource list is empty and there's no search query.".

It's more verbose, but I think it makes it more clear what `NoResources` is used for.